apps/dashboard/src/components/activity/components/activity-overview.tsx

```diff
@@ -37,12 +38,17 @@ export function ActivityOverview({ activity }: ActivityOverviewProps) {
 
         <OverviewItem label="Transaction ID" value={activity.transactionId} isCopyable />
 
-        <OverviewItem
-          label="Subscriber ID"
-          isDeleted={!activity.subscriber}
-          value={(activity.subscriber?.subscriberId || activity._subscriberId) ?? ''}
-          isCopyable
-        />
+        <SubscriberDrawerButton
+          className="text-start"
+          subscriberId={activity.subscriber?.subscriberId || activity._subscriberId}
+        >
+          <OverviewItem
+            label="Subscriber ID"
+            isDeleted={!activity.subscriber}
+            value={(activity.subscriber?.subscriberId || activity._subscriberId) ?? ''}
+            isCopyable
```

Comment:
I can't copy the Subscriber ID :/ 
IMO the label should be clickable and open the drawer, not the value, it's super confusing.

https://github.com/user-attachments/assets/bedbf79c-24b5-4090-b068-d7ea73052cd3